.github/workflows/release-discover-branches.yaml

```diff
@@ -39,6 +39,10 @@ jobs:
         working-directory: ./src/github.com/openshift-knative/hack
         run: make discover-branches ARGS="--config config/"
 
+      - name: Generate Action
+        working-directory: ./src/github.com/openshift-knative/hack
+        run: make generate-ci-action
```

Comment:
cmd/discover seems to call this already: https://github.com/openshift-knative/hack/blob/65af24477502773b06320352d29118e1ae1f2d17/pkg/discover/discover.go#L46-L50

:thinking:

Reply:
Good catch, fixed